addons/html_editor/static/src/others/embedded_components/core/table_of_content/table_of_content.xml

```diff
@@ -1,16 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates id="template" xml:space="preserve">
 <t t-name="html_editor.EmbeddedTableOfContent">
-    <div class="o_embedded_toc_content">
-        <t t-foreach="this.state.toc.headings" t-as="heading" t-key="heading_index">
-            <a href="#" contenteditable="false"
-                t-out="heading.name"
-                t-on-click.prevent="() => this.onTocLinkClick(heading)"
-                t-attf-class="o_no_link_popover py-1 pe-1 d-block o_embedded_toc_link #{'o_embedded_toc_link_depth_' + heading.depth}"/>
-        </t>
+    <div class="px-1" t-att-class="{ 'pb-1' : state.open and state.toc.headings.length }">
```

Comment:
```suggestion
    <div class="px-1" t-att-class="{ 'pb-1' : state.open or state.toc.headings.length lt 2 }">
```
because we need a padding when there is less than 2 headings, since the hint `Add headings to fill the Table of Contents` appears, even when closed, and without the padding bottom, there is not enough space below that text.

Reply:
I think doing so will make the "Table of Content" title look a little above the center for the toc having headings < 2 and the user with read access.

To handle both these issues, what if we take the hint outside of the wrapper div?